data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,88 @@ mission "FW Epilogue: Danforth"
+			`	"I'm not sure about this, what would you have me do?" you say.`
+			`	"Not much out of the ordinary from our tests in the past," he says. "You'll find a ship of ours in orbit that goes by 'Hunk', it has been fitted with a few of these new weapons. Your job will be to fight it and disable it, then tell us what you think of its combat capabilities and transfer your ships logs so we can analyze the fight."`
+			choice
+				`	"Alright, I suppose for old times sake I can help out."`
+				`	"Sorry, I just wanted to check in and see how you were doing."`
+					goto catchup
+			`	He turns to you and says with a smirk, "Now mind you, we just want you to disable the ship, not destroy it. We've recommissioned it from retirement, so it's not top of the line but all the same I want to be able to keep using it for future tests. Understand?"`
+			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`
+				accept
+			label catchup
+			`	"I see, well I'm sure we can find someone else up to the task. Regardless captain <last>, I assure you this new weapon vastly improves on the outdated Plasma Cannon, this will put Kraz Cybernetics in the big leagues!"`
+			`	As his tangent about his new invention continues, you think to yourself about the old times and the war, and all the people you've met along the way.`
+			`	"How about we get out of here <first>? My work here is done today, I know a homely little bar not far from here."`
+			choice
+				`	"Of course, it's not like there's a war to worry about anymore."`
+			`"Ha well of course, the war kept all of us plenty busy." He leads you out of the facility, all the while telling an anecdote that must be humor for an engineer.`
+			`	You follow Barmy into this bar he was telling you of, even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks, Barmys order catches you off guard and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle and you both tell stories of your early lives, specifically before leaving the ground. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
+			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there, I was lucky to get out myself"`
+			`	"No way you're from New Boston too! What are the chances?" you hastily respond.`
```

Comment:
```suggestion
			`	"No way you're from New Boston too! What are the chances?" you respond.`
```
`hastily` implies some sort of rushed/frantic sense, which doesn't seem appropriate for the conversation.